From 8d34278cc99275cdfcc7503f84b0d39f402024ce Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Wed, 26 Aug 2015 19:21:00 +0200
Subject: kbuild: Enable DT symbols when CONFIG_OF_OVERLAY is used

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
---
Replacement for "ARM: DT: Enable symbols when CONFIG_OF_OVERLAY is
used".
---
 scripts/Makefile.lib | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 1ca4dcd..c50473b 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -298,6 +298,10 @@ cmd_dt_S_dtb=						\
 $(obj)/%.dtb.S: $(obj)/%.dtb
 	$(call cmd,dt_S_dtb)
 
+ifeq ($(CONFIG_OF_OVERLAY),y)
+DTC_FLAGS += -@
+endif
+
 quiet_cmd_dtc = DTC     $@
 cmd_dtc = mkdir -p $(dir ${dtc-tmp}) ; \
 	$(CPP) $(dtc_cpp_flags) -x assembler-with-cpp -o $(dtc-tmp) $< ; \
-- 
cgit v1.1

